rpm/gvisor-tap-vsock.spec

```diff
@@ -92,8 +92,8 @@ CGO_CFLAGS=$(echo $CGO_CFLAGS | sed 's/-specs=\/usr\/lib\/rpm\/redhat\/redhat-an
 export CGO_CFLAGS+=" -m64 -mtune=generic -fcf-protection=full"
 %endif
 
-# reset LDFLAGS for plugins and gvisor binaries
-LDFLAGS=''
+# reset LDFLAGS for plugins and gvisor binaries, but ensure gvproxy --version and such is set
+LDFLAGS="-X github.com/containers/gvisor-tap-vsock/pkg/types.PackageVersion=%{version}"
```

Comment:
this change is removed in the last commit, maybe something that got added for testing?

Reply:
In the first commit, the only way to set the version is by using `-X github.com/containers/gvisor-tap-vsock/pkg/types.PackageVersion` (or rather `gitVersion` - I've fixed it). This is added to the spec file as otherwise the version in the binaries it builds will be quite wrong. 

The 3rd commit sets the version automatically in tarballs, so we no longer need to explicitly use `-X github.com...` in the spec file, so it's removed.